### PR TITLE
Fix collections

### LINKED ISSUE #1980: bugs with create / edit collection page
1. Putting anything in location, thumbnail, representative, and related URL fields throws exception NoMethodError when attempting to save.

2. Title is a required field but it isn’t asterisked

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #1980 
Remove thumbnail and representative.  Add based_near and related_url to the collection presenter.
@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: app/forms/sufia/forms/collection_form.rb [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+       :tag,
+       :rights,
+       :publisher,
+       :date_created,
+       :subject,
+       :language,
+       :identifier,
+       :based_near,
+       :related_url,
+       :resource_type]
```
Comment (mjgiarlo) [inline app/forms/sufia/forms/collection_form.rb]: I'm not sure collections should have resource types, but I don't care strongly about this.

File: spec/presenters/sufia/collection_presenter_spec.rb [hunk 2]
```diff
@@ -8,7 +8,12 @@
                             :language, :identifier, :based_near, :related_url] }
   end
 
-  let(:collection) { build(:collection, id: '111', description: ['a nice collection'], title: ['A clever title'], resource_type: ['Collection']) }
+  let(:collection) { build(:collection, # id: '111',
```
Comment (mjgiarlo) [inline spec/presenters/sufia/collection_presenter_spec.rb]: Remove cruft?

File: app/forms/sufia/forms/collection_form.rb (hunk 1 above)
Comment (mjgiarlo) [inline app/forms/sufia/forms/collection_form.rb]: Meh, I take it back.

Comment (jcoyne) [inline app/forms/sufia/forms/collection_form.rb]: I'd love to remove resource_type if you're game.

Comment (mjgiarlo) [inline app/forms/sufia/forms/collection_form.rb]: I am game if you think it's OK to include that in this PR.

**Comment by jcoyne** [comment]
@mjgiarlo not this PR, but the next.

**Comment by mjgiarlo** [comment]
@jcoyne :ok_hand:

**Pull request merged**